### ISSUE TITLE
Feature Request: Beam.pro support

### ISSUE THREAD

**Issue by skwerlman** [Gyoo/Discord-Streambot#28]
https://beam.pro/

https://developer.beam.pro/

Beam.pro is a relatively new streaming service, but it already has a pretty good community. They already provide a Java wrapper for their API, so hopefully that'll make adding support a bit easier.

**Comment by p455w0rd** [comment]
Definitely need this :D

**Comment by Officialy** [comment]
Yes! Definitely need this when I go live, would be helpful for many others too :D
